lib/utils/config.ts

```diff
@@ -13,77 +13,60 @@ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 See the License for the specific language governing permissions and
 limitations under the License.
 */
-import * as fs from 'fs';
-
 import Promise = require('bluebird');
 import ResinSdk = require('resin-sdk');
-import deviceConfig = require('resin-device-config');
 import * as semver from 'resin-semver';
 
 const resin = ResinSdk.fromSharedOptions();
 
-function readRootCa(): Promise<string | void> {
-	const caFile = process.env.NODE_EXTRA_CA_CERTS;
-	if (!caFile) {
-		return Promise.resolve();
-	}
-	return Promise.fromCallback(cb =>
-		fs.readFile(caFile, { encoding: 'utf8' }, cb),
-	)
-		.then(pem => Buffer.from(pem).toString('base64'))
-		.catch({ code: 'ENOENT' }, () => {});
-}
+type ImgConfig = {
```

Comment:
You're stop importing `device-config` but aren't removing it as a dependency so I think you should use the type from that module instead of duplicating it here.

Reply:
@dfunckt I might be missing something, but I think we didn't have typings for `resin-device-config` and it was just `any`.

Reply:
@dfunckt  I could of course try moving this in this custom repo-wise typings for `resin-device-config` and see whether anything else breaks if you prefer that.

Reply:
That sounds like a good idea. That module is also used in the api. I would not expect anything to break, adding types means TS will just not compile and we’ll fix it and CS code will just ignore it.